plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/wrapLayerManager.cpp

```diff
@@ -15,32 +15,17 @@
 //
 #include "AL/usdmaya/nodes/LayerManager.h"
 
-#include <boost/python/args.hpp>
-#include <boost/python/def.hpp>
-
-// On Windows, against certain versions of Maya and with strict compiler
-// settings on, we are getting warning-as-error problems with a couple
-// boost includes.  Disabling those warnings for the specific includes
-// for now instead of disabling the strict settings at a higher level.
-#ifdef _MSC_VER
-#pragma warning(push)
-#pragma warning(disable : 4275)
-#endif
```

Comment:
I'm hoping this is no longer needed. Other files include this header without problems.